functionnal_tests/cypress/integration/account/translation.js

```diff
@@ -0,0 +1,20 @@
+import { PAGES } from '../../support/urls_commands.js'
```

Comment:
Unused import

Reply:
It is, now. I used visitPage whenever possible. Thanks!